### PR TITLE
testing/qt-creator: upgrade to 4.9.0

### PR DESCRIPTION
**Pull request by maxice8**

### CODE REVIEW

**Comment by maxice8** [comment]
@ncopa one less Qt4 dependency

**Comment by maxice8** [comment]
ping

File: testing/qt-creator/APKBUILD [hunk 1]
```diff
@@ -1,62 +1,50 @@
...
-source="http://download.qt-project.org/official_releases/qtcreator/${pkgver%.*}/$pkgver/qt-creator-opensource-src-$pkgver.tar.gz
-	qtcreator-arm.patch
-	fix-headers-musl.patch
+arch="all !ppc64le !s390x"
+license="GPL-3.0-only WITH Qt-GPL-exception-1.0"
+depends="
+	qt5-qtbase-sqlite
+	qt5-qtquickcontrols
+	qt5-qtquickcontrols2
+	qt5-qtbase-dev
```
Comment (PureTryOut) [inline testing/qt-creator/APKBUILD]: This doesn't seem right, shouldn't it belong in `$makedepends`?

**Comment by maxice8** [comment]
QtCreator is a Qt5 IDE so it needs the basic qt5 tooling to work, stuff
like qmake

**Comment by PureTryOut** [comment]
Ah of course, never mind!

**Pull request closed without merging**